edge-hub/core/test/Microsoft.Azure.Devices.Routing.Core.Test/MessageQueueIdHelperTest.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Routing.Core.Test
+{
+    using System;
+    using Microsoft.Azure.Devices.Edge.Util.Test.Common;
+    using Xunit;
+
+    public class MessageQueueIdHelperTest
+    {
+        [Fact]
+        [Unit]
```

Comment:
nit: You can add [Unit] attribute at the top of the class to have it apply to the whole class

Reply:
done.